Conceptpower+Spring/src/test/java/edu/asu/conceptpower/core/impl/ConceptListManagerTest.java

```diff
@@ -0,0 +1,78 @@
+package edu.asu.conceptpower.core.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.conceptpower.core.ConceptList;
+import edu.asu.conceptpower.db4o.IConceptDBManager;
+
+public class ConceptListManagerTest {
+
+    @Mock
+    private IConceptDBManager client = Mockito.mock(IConceptDBManager.class);;
+
+    @InjectMocks
+    private ConceptListManager conceptListManager;
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+
+        List list = new ArrayList();
+        ConceptList conceptList = new ConceptList();
+        conceptList.setConceptListName("First List");
+        conceptList.setDescription("First List Description");
+        list.add(conceptList);
+
+        Mockito.when(client.getAllElementsOfType(ConceptList.class)).thenReturn(list);
+        Mockito.when(client.getConceptList("First List")).thenReturn(conceptList);
+
+    }
+
+    @Test
+    public void getAllConceptListsTest() {
+        List list = conceptListManager.getAllConceptLists();
+        assertNotNull(list);
+        assertEquals(1, list.size());
+    }
+
+    @Test
+    public void getConceptListTest() {
+        String listName = "First List";
+        ConceptList conceptList = conceptListManager.getConceptList(listName);
+        assertNotNull(conceptList);
+        assertEquals(listName, conceptList.getConceptListName());

```

Comment:
instead of this, I would test for equality of the returned concept list and the concept list you defined in the init method